contracts/ROSCA.sol

```diff
@@ -175,6 +238,7 @@ contract ROSCA {
     if (distrubtionAmountInWei >= lowestBid ||
```

Comment:
can we rename "distributionAmountInWei" to "bidInWei" for clarity (as we're comparing to lowestBid later)?